storage/src/db/kv_db/tables.rs

```diff
@@ -418,10 +411,27 @@ impl ColumnFamily for InternalMessageStatistics {
 
 impl ColumnFamilyOptions<Caches> for InternalMessageStatistics {
     fn options(opts: &mut Options, caches: &mut Caches) {
-        zstd_block_based_table_factory(opts, caches);
+        internal_queue_options(opts, caches);
     }
 }
 
+fn internal_queue_options(opts: &mut Options, caches: &mut Caches) {
+    let mut block_factory = BlockBasedOptions::default();
+    block_factory.set_block_cache(&caches.block_cache);
+    block_factory.set_format_version(6);
+
+    opts.set_block_based_table_factory(&block_factory);
+    opts.set_disable_auto_compactions(true);
+    opts.set_compression_type(DBCompressionType::None);
+
+    opts.set_level_compaction_dynamic_level_bytes(true);
+
+    // optimize for bulk inserts and single writer
+    opts.set_max_write_buffer_number(8); // 8 * 512MB = 4GB
+    opts.set_min_write_buffer_number_to_merge(2); // allow early flush
+    opts.set_write_buffer_size(256 * 1024 * 1024); // 512 per memtable
```

Comment:
not sure if you need it with a separate db. You have pretty low traffic. Start with 2 x 256